pymkv/MKVTrack.py

```diff
@@ -468,17 +482,27 @@ def extract(self, output_path: str | os.PathLike | None = None, silent: bool | N
             extract_info_file += f"_{self.language}"
         if self.extension:
             extract_info_file += f".{self.extension}"
-        if (not self.language and not self.expansion) and self.track_name:
```

Comment:
Replaced this with `self.extension`

Reply:
Yeah, that's how it's supposed to be. There was a typo in the variable name.